src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -511,7 +516,7 @@ export default {
 }
 
 .loan-card-active-hover:hover .loan-card-use {
-	@apply tw-underline;
+	@apply tw-no-underline;
```

Comment:
I think `loan-card-active-hover` can be removed entirely.

Reply:
Removed